tools/reach-calculator.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Site Breakage Reach Calculator</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="stylesheet" href="https://duckduckgo.com/duckduckgo-help-pages/css/main.css">
+    <link rel="icon" type="image/ico" href="https://duckduckgo.com/duckduckgo-help-pages/favicon.ico">
+    <link rel="icon" type="image/svg+xml" sizes="any" href="https://duckduckgo.com/duckduckgo-help-pages/favicon.svg">
+    <link rel="apple-touch-icon" href="https://duckduckgo.com/duckduckgo-help-pages/apple-touch-icon.png">
+    <!-- Enable support for HTML5 elements (e.g. <nav>, <footer>) -->
+    <!--[if lt IE 9]>
+        <script src="https://duckduckgo.com/duckduckgo-help-pages/scripts/html5shiv.js"></script>
+    <![endif]-->
```

Comment:
I think we can safely drop support for IE9 here :)

```suggestion
```

Reply:
Fair point – done.